hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/DispatchForResponseCodeHtsCall.java

```diff
@@ -219,6 +219,7 @@ public DispatchForResponseCodeHtsCall(
         if (status != SUCCESS) {
             status = failureCustomizer.customize(syntheticBody, status, enhancement);
             recordBuilder.status(status);
+            return reversionWith(gasRequirement, recordBuilder);
```

Comment:
yikes.

Reply:
According to @tinker-michaelj , the expected behavior is to not revert.  Took that change out.

Reply:
Ummm, OK if he says so.  But what's the rule?